ocdaction/ocdaction/templates/dashboard/act/task_edit.html

```diff
@@ -5,7 +5,15 @@
 
 {% block main %}
 
-
+    <script language="javascript">
+        function checkCancel() {
+            if (confirm("Are you sure you want to discard changes?")) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+    </script>
```

Comment:
Can we move the script the bottom of the page? (as per: https://developer.yahoo.com/performance/rules.html#js_bottom)